x-pack/plugins/security/server/authentication/index.ts

```diff
@@ -137,17 +137,19 @@ export async function setupAuthentication({
 
   authLogger.debug('Successfully registered core authentication handler.');
 
+  const apiKeys = new APIKeys({
+    clusterClient,
+    logger: loggers.get('api-key'),
+    isSecurityFeatureDisabled,
+  });
   return {
     login: authenticator.login.bind(authenticator),
     logout: authenticator.logout.bind(authenticator),
     getCurrentUser,
-    createAPIKey: (request: KibanaRequest, body: CreateAPIKeyOptions['body']) =>
-      createAPIKey({
-        body,
-        loggers,
-        isSecurityFeatureDisabled,
-        callAsCurrentUser: clusterClient.asScoped(request).callAsCurrentUser,
-      }),
+    createAPIKey: (request: KibanaRequest, body: CreateAPIKeyParams) =>
```

Comment:
__nit:__ in some places we use `body`, in others `params`, let's pick one name :) I vote for `params` since the types end with `Params` already.